desktop/libs/indexer/src/indexer/indexers/phoenix_sql.py

```diff
@@ -69,22 +72,30 @@ def create_table_from_file(self, source, destination, start_time=-1, dry_run=Fal
           'primary_keys': ', '.join(destination.get('indexerPrimaryKey'))
       }
 
-    if destination['indexerRunJob']:
-      sql += '''
-  UPSERT INTO %(table_name)s VALUES ('NY','New York',8143197);
-  UPSERT INTO %(table_name)s VALUES ('CA','Los Angeles',3844829);
-  UPSERT INTO %(table_name)s VALUES ('IL','Chicago',2842518);
-  UPSERT INTO %(table_name)s VALUES ('TX','Houston',2016582);
-  UPSERT INTO %(table_name)s VALUES ('PA','Philadelphia',1463281);
-  UPSERT INTO %(table_name)s VALUES ('AZ','Phoenix',1461575);
-  UPSERT INTO %(table_name)s VALUES ('TX','San Antonio',1256509);
-  UPSERT INTO %(table_name)s VALUES ('CA','San Diego',1255540);
-  UPSERT INTO %(table_name)s VALUES ('TX','Dallas',1213825);
-  UPSERT INTO %(table_name)s VALUES ('CA','San Jose',91233);
-  ''' % {
-          'table_name': table_name,
-        }
+    source_path = urllib_unquote(source['path'])
```

Comment:
Move all the new phoenix code into a phoenix module? (like for the other dialect)

Reply:
meaning? it is already in a separate file.